users/tests/test_unit_customuser_model.py

```diff
@@ -64,6 +64,11 @@ def test_user_queryset_active_method_should_return_queryset_of_only_active_users
         self.assertTrue(active_user in queryset)
         self.assertFalse(inactive_user in queryset)
 
+    def test_full_clean_method_should_be_called_on_save(self):
+        user = CustomUser()
+        with self.assertRaises(ValidationError):
+            user.save()
```

Comment:
What you test here is only that `save` method raise `ValidationError`. You haven't test that `full_clean` method is called. Please add `patch` for `full_clean` method and test that method was called ideally once without any arguments.

Reply:
Done. I use mock and check `assert_called_once()` now.